.github/workflows/push.yml

```diff
@@ -51,15 +51,15 @@ env:
   APPLE_APP_IDENTITY: "Developer ID Application: Tomas Netrval"
 
 jobs:
-  set_version:
+  set-version:
     runs-on: ubuntu-latest
     outputs:
-      tag: ${{ steps.set_version.outputs.tag }}
-      is_semantic_tag: ${{ steps.set_version.outputs.is_semantic_tag }}
-      version: ${{ steps.set_version.outputs.version }}
+      tag: ${{ steps.set-version.outputs.tag }}
+      is_semantic_tag: ${{ steps.set-version.outputs.is_semantic_tag }}
+      version: ${{ steps.set-version.outputs.version }}
```

Comment:
@JakubMatejka nevyhodnoti to GHA ako `-`? Myslim, ze preto tam bolo `_`, ale mozno sa mylim.

Reply:
Tak asi je to OK. Sorry (niekde s tym bol problem, a uz neviem kde :slightly_smiling_face:  )

![image](https://user-images.githubusercontent.com/19371734/154275078-475eeb82-d2e5-4da9-889d-6b25803d7788.png)

Reply:
Taky jsem to čekal, ale prošlo to. (Chtěl jsem teda původně změnit všechno na podtržítka, ale pomlček bylo míň. 😃 Ale je to jedno, klidně podtržítka jestli se ti líbí víc. Chtěl jsem hlavně aby se to sjednotilo.)

Reply:
Je mi to uplne jedno, super ze to bude jednotne. Len som sa obaval toho `minus`.